crates/core/src/db/datastore/system_tables.rs

```diff
@@ -715,6 +746,43 @@ impl From<StConstraintRow> for ConstraintSchema {
     }
 }
 
+/// System Table [ST_ROW_LEVEL_SECURITY_NAME]
+///
+/// | row_level_security_id | row_level_security_name | table_id | sql          |
+/// |-----------------------|-------------------------|----------|--------------|
+/// | 1                     | "rls_test"              | 1        | "SELECT ..." |
+#[derive(Debug, Clone, PartialEq, Eq, SpacetimeType)]
+#[sats(crate = spacetimedb_lib)]
+pub struct StRowLevelSecurityRow {
+    pub(crate) row_level_security_id: RowLevelSecurityId,
+    pub(crate) row_level_security_name: Box<str>,
+    pub(crate) table_id: TableId,
+    pub(crate) sql: Box<str>,
```

Comment:
I was wondering if we might want to add another field here storing a cache for a compiled expression of some sort, but on further reflection I think that's liable to cause problems, better to have the system tables be less redundant.